### ISSUE TITLE
All sites except Gent: let windows users check ssh fingerprint

### ISSUE THREAD

**Issue by redfast00** [hpcugent/vsc_user_docs#228]
Currently, in all sites except Gent, the Windows HPC-intro doesn't list ssh fingerprints, so users blindly hit "Yes". This can be a security issue, so please fix this after #227 gets merged.

**Comment by wpoely86** [comment]
Instead of hoping that users will look at this before their first time connecting, we should better just publish SSHFP records (with DNSSEC)

**Comment by boegel** [comment]
@wpoely86 And until then you think it's OK to just tell him to answer "`yes`"?

**Comment by wpoely86** [comment]
Do you honestly think any users has ever looked up the fingerprints on first connect?

**Comment by boegel** [comment]
I don't know, maybe some have...

It's not because people won't follow instructions that we should refrain from giving correct instructions. ;)

**Comment by wpoely86** [comment]
As long as we don't kid our selves that having them in the user docs increase security.

I've added `SSHFP` records to our DNS for `login.hpc.vub.ac.be`, DNSSEC should also be added but that's beyond my rights on the DNS server.

Our fingerprints are:
```
2048 SHA256:xunjQm9tWulj3hkmSf/3EGs7cOThFIa+Zgn2AZyTtbc /etc/ssh/ssh_host_rsa_key.pub (RSA)
1024 SHA256:5gWMMTRS6AwTy3Qx2qZEjnkxXbK/FNjU0jZFlr1Enys /etc/ssh/ssh_host_dsa_key.pub (DSA)
256 SHA256:FJmw0a3ko64k3cQXECAC7R9BnaN/g3BOFjnLARzQhVQ /etc/ssh/ssh_host_ecdsa_key.pub (ECDSA)
```

**Comment by boegel** [comment]
@redfast00 Can this be closed thanks to #275?
